fail2ban/server/action.py

```diff
@@ -532,7 +532,7 @@ def _processCmd(self, cmd, aInfo = None):
 		return self.executeCmd(realCmd, self.timeout)
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60):
+	def executeCmd(realCmd, timeout=60, testCmd=False):
```

Comment:
Because some branch lines will be not executed during test cases, but in production (and possible makes boom).
Counter question: why you will exclude seek/log for the test case?